internal/engine/experiment/stunreachability/stunreachability_test.go

```diff
@@ -1,67 +1,82 @@
-package stunreachability_test
+package stunreachability
 
 import (
 	"context"
 	"errors"
 	"net"
-	"os"
 	"strings"
 	"testing"
 
 	"github.com/apex/log"
-	"github.com/ooni/probe-cli/v3/internal/engine/experiment/stunreachability"
 	"github.com/ooni/probe-cli/v3/internal/engine/mockable"
 	"github.com/ooni/probe-cli/v3/internal/engine/model"
 	"github.com/ooni/probe-cli/v3/internal/netxlite"
 	"github.com/pion/stun"
 )
 
+const (
+	defaultEndpoint = "stun.ekiga.net:3478"
+	defaultInput    = "stun://" + defaultEndpoint
+)
+
 func TestMeasurerExperimentNameVersion(t *testing.T) {
-	measurer := stunreachability.NewExperimentMeasurer(stunreachability.Config{})
+	measurer := NewExperimentMeasurer(Config{})
 	if measurer.ExperimentName() != "stunreachability" {
 		t.Fatal("unexpected ExperimentName")
 	}
-	if measurer.ExperimentVersion() != "0.2.0" {
+	if measurer.ExperimentVersion() != "0.3.0" {
 		t.Fatal("unexpected ExperimentVersion")
 	}
 }
 
-func TestRun(t *testing.T) {
-	if os.Getenv("GITHUB_ACTIONS") == "true" {
-		// See https://github.com/ooni/probe-engine/issues/874#issuecomment-679850652
-		t.Skip("skipping broken test on GitHub Actions")
```

Comment:
Skipping is not necessary anymore because this whole suite now uses ekiga.net as the default to test.